network-api/networkapi/nav/models.py

```diff
@@ -151,27 +151,37 @@ def blog_index_page(self):
 
     @property
     def localized_featured_blog_topics(self):
+        en_locale = wagtail_models.Locale.objects.get(language_code="en")
+        active_locale = wagtail_models.Locale.get_active()
+
+        # Get localized blog_topics
         featured_topics_relationships = self.featured_blog_topics.all().select_related("topic", "icon")
```

Comment:
Since this already retrieves a localized list of topics, we don't need to build out a cache of localized topics, but we can still implement that technique for the `en` locale to keep this performing well, rather than something like translating the slug each time it's called.